paper/paper.md

```diff
@@ -58,13 +58,10 @@ sources is aimed at, for validation reasons.
 
 The methods and operators for the description of linear elasticity and material modeling are mathematically simple and can be expressed in a compact manner.
 Due to this relative simplicity, they are usually implemented independently by each scientist for use in their studies.
-Consequentially, unlike in other research field such as physics and computer science, there is no library for the common methods and operators of linear elasticity and modeling to this day.
-Apart from didactic advantages, this procedure causes problems for
-the exchange of research code and
-the reliability of the implementations.
-This presents a major obastacle with regard to the exchange and reliability of research code as well as the interpretation and comparison of results.
+Consequentially, unlike in other research fields such as physics and computer science, there is no library for the common methods and operators of linear elasticity and modeling to this day.
+This presents a major obstacle with regard to the exchange and reliability of research code as well as the interpretation and comparison of results.
 
-The main goal of the project `mechkit` is reusable research code that increases the reliability of the research results and
+The main goal of `mechkit` is reusable research code that increases the reliability of results and
```

Comment:
"the project" has not been used in conjunction with `mechkit` before in this publication